### PR TITLE
Prusa3D.PrusaSlicer version 2.5.0

### PR DESCRIPTION
**Pull request by s-crypt**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/78975)

### CODE REVIEW

**Comment by s-crypt** [comment]
Man, there will be one day that I get it right on the first try, but today is not that day.

**Comment by s-crypt** [comment]
Hmm, this is strange. When I install locally I do not get any errors and get a proper silent install. This is the same installer and options as previous versions that did not have this problem. I am not quite sure what is going on.

**Comment by s-crypt** [comment]
@OfficialEsco or @ItzLevvie would it be possible to get some assistance with this one? I am not getting errors that the pipeline is logging, or I don't fully understand the problem.

File: manifests/p/Prusa3D/PrusaSlicer/2.5.0/Prusa3D.PrusaSlicer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```
Comment (OfficialEsco) [inline manifests/p/Prusa3D/PrusaSlicer/2.5.0/Prusa3D.PrusaSlicer.installer.yaml:18]: I’ve not checked the logs but could it be this?
```suggestion
- Architecture: x64
```

File: manifests/p/Prusa3D/PrusaSlicer/2.5.0/Prusa3D.PrusaSlicer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://www.prusa3d.com/downloads/drivers/PrusaSlicer_Win_standalone_2.5.0.exe
+  InstallerSha256: 1A417FE95D2EEBD7E0D5A4A2A51E043BA594A865F223841A45CA6D4E88E9444C
```
Comment (OfficialEsco) [inline manifests/p/Prusa3D/PrusaSlicer/2.5.0/Prusa3D.PrusaSlicer.installer.yaml:21]: Maybe it was cached?

```
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   F97CAAE8172E3204538E855EE9A32F04FC41BE0D67FF7E54C0C903E6DC1DBC26
Expected hash: 1a417fe95d2eebd7e0d5a4a2a51e043ba594a865f223841a45ca6d4e88e9444c
```
```
Get-FileHash ./PrusaSlicer_Win_standalone_2.5.0.exe

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          F97CAAE8172E3204538E855EE9A32F04FC41BE0D67FF7E54C0C903E6DC1DBC26
```

```suggestion
  InstallerSha256: F97CAAE8172E3204538E855EE9A32F04FC41BE0D67FF7E54C0C903E6DC1DBC26
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
I've also just checked the logs from Checks -> 0 errors / 1 warnings -> Artifacts: 3 Published -> Download InstallationVerificationLogs

From the logs/screenshot there is no indicator, but there is a window that we don't see so it might be a warning about not supported on x86, we'll see after the hash change =)

**Comment by s-crypt** [comment]
@OfficialEsco That seems to have fixed the problem. Thanks so much!

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**